core/modules/savers/put.js

```diff
@@ -21,15 +21,17 @@ Select the appropriate saver module and set it up
 var PutSaver = function(wiki) {
 	this.wiki = wiki;
 	var self = this;
+	var uri = encodeURI(document.location.protocol + "//" + document.location.hostname + ":" + document.location.port + document.location.pathname);
 	// Async server probe. Until probe finishes, save will fail fast
```

Comment:
I opted not to extend the fail-fast check below to include the ETag to ensure that remains optional. (While I'm guessing all WebDAV servers support this, I'm not actually sure about that.)